### PR TITLE
feat(bundler): Add Spanish and SpanishInternational to NSIS languages (closes #6852)

### LINKED ISSUE #6852: [feat] Add Spanish to the available languages (Windows NSIS)
### Describe the problem

Spanish is one of the listed languages on https://github.com/kichik/nsis/tree/9465c08046f00ccb6eda985abbdbf52c275c6c4d/Contrib/Language%20files, yet it doesn't seem to be available for the Tauri NSIS bundle. Large-scale applications made with Tauri may need to have Spanish as an available language on the installer.

```
Error failed to bundle project: Language Spanish not implemented. If it is a valid language listed on
<https://github.com/kichik/nsis/tree/9465c08046f00ccb6eda985abbdbf52c275c6c4d/Contrib/Language%20files>, please open a
Tauri feature request
```

### Describe the solution you'd like

Spanish should be added to the list of available languages for the Windows installer. The language files are available on https://github.com/kichik/nsis/tree/9465c08046f00ccb6eda985abbdbf52c275c6c4d/Contrib/Language%20files.

### Alternatives considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by luisfi-dev**
Spanish is one of the listed languages on https://github.com/kichik/nsis/tree/9465c08046f00ccb6eda985abbdbf52c275c6c4d/Contrib/Language%20files, yet it's not available for the Tauri NSIS bundle. Large-scale applications made with Tauri may need to have Spanish as an available language on the installer.

### What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Docs
- [ ] New Binding issue #___
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

### Does this PR introduce a breaking change?

- [ ] Yes, and the changes were approved in issue #___
- [x] No

### Checklist

- [x] When resolving issues, they are referenced in the PR's title (e.g `fix: remove a typo, closes #___, #___`)
- [x] A change file is added if any packages will require a version bump due to this PR per [the instructions in the readme](https://github.com/tauri-apps/tauri/blob/dev/.changes/readme.md).
- [x] I have added a convincing reason for adding this feature, if necessary

### CODE REVIEW

**Comment by FabianLars** [comment]
Thanks for contributing!

Two small things are missing:
1) These langs also have to be added here: https://github.com/tauri-apps/tauri/blob/dev/tooling/bundler/src/bundle/windows/nsis.rs#LL505C13-L505C14
2) We need a changefile similar to https://github.com/tauri-apps/tauri/blob/dev/.changes/nsis-webview-installmodes.md

**Comment by luisfi-dev** [comment]
Working on it

**Comment by FabianLars** [comment]
Thanks! Also i _think_ it's fine to add languages in a `patch` instead of `minor` but i let the others be the judge of that.

File: .changes/nsis-languages.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+'tauri-bundler': 'minor'
```
Comment (wusyong) [inline .changes/nsis-languages.md]:
```suggestion
"tauri-bundler": patch
```

@FabianLars I based on your comment to do this. It should be more clear for the author.
Good review btw!

**Comment by FabianLars** [comment]
Okay last thing and then we should be done: Can you run `cargo fmt` once? 🙏

**Comment by luisfi-dev** [comment]
Done. Exited with code `0` and no files were modified, so I think we're good?

**Comment by FabianLars** [comment]
hmm, ci complains about the nsis.rs file and it indeed looks off. Did you try to `cd` into the tooling/bundler/ folder before running `cargo fmt`?

**Comment by luisfi-dev** [comment]
Ran again from `/tooling/bundler`. Exited with code `0`, but this time it corrected the indentation for  the `nsis.rs` file. Will commit the changes.

**Review by FabianLars — APPROVED** [review]
Thank you! ❤️

**Comment by luisfi-dev** [comment]
Crap... I forgot to sign the commit. I'll re-fork the repo and submit another PR.

**Comment by FabianLars** [comment]
This one worked for me in the past to sign older commits when i messed something up: https://superuser.com/a/1123928 a bit less work than redoing it from scratch.

**Comment by luisfi-dev** [comment]
That is exactly what I tried, didn't work for me.

**Pull request closed without merging**